gatk-launch

```diff
@@ -23,14 +23,17 @@ BUILD_LOCATION = script +"/build/install/" + projectName + "/bin/"
 GATK_RUN_SCRIPT = BUILD_LOCATION + projectName
 BIN_PATH = script + "/build/libs"
 
+EXTRA_JAVA_OPTIONS="-Dsamjdk.intel_deflater_so_path=libIntelDeflater.so -Dsamjdk.compression_level=1 -DGATK_STACKTRACE_ON_USER_EXCEPTION=true "
```

Comment:
we probably don't want DGATK_STACKTRACE_ON_USER_EXCEPTION on by default, do we? (maybe we do for the alpha period)